src/victory-primitives/clip-path.js

```diff
@@ -28,28 +29,25 @@ export default class ClipPath extends React.Component {
     translateY: PropTypes.number
   };
 
+  static defaultProps = {
+    translateX: 0,
```

Comment:
I don't want these as default props because component props are used as defaults in parent components like VictoryTransition and VictoryZoomContainer. Please remove these and return to falling back to `val || 0` in the code.